nmosnode/api.py

```diff
@@ -28,6 +28,7 @@
 NODE_REGVERSION = _config.get('nodefacade', {}).get('NODE_REGVERSION', 'v1.2')
 NODE_APINAMESPACE = "x-nmos"
 NODE_APINAME = "node"
+DEVICE_ROOT = '/' + NODE_APINAMESPACE + '/' + NODE_APINAME + '/'
```

Comment:
Nit picking, but could we use something other than 'device' given its usage within the NMOS data model? `NODE_APIROOT` may be better?